dependencies.json

```diff
@@ -1,4 +1,8 @@
 [ {
+  "module_name" : "OntologyAPI",
+  "type" : "sdk",
+  "version_tag" : "dev"
```

Comment:
Is there any reason we can't release the ontology API at this point? I'd prefer not to check in a hardcoded dev client.